server.py

```diff
@@ -186,32 +173,11 @@ def show_user_tasks_and_gameplan():
 
     gameplan_tasks = GameplanTask.query.filter_by(user_id=user_id).order_by(GameplanTask.order).all()
     tasks = Task.query.filter_by(user_id=user_id).all()
-
-    priority_object = UserSetting.query.filter_by(user_id=user_id,
-                                                  setting_id=1).first()
-    if priority_object:
-        priority = priority_object.value
-    else:
-        priority = None
-
-    intention_object = UserSetting.query.filter_by(user_id=user_id,
-                                                   setting_id=9).first()
-    if intention_object:
-        intention = intention_object.value
-    else:
-        intention = None
-
-    notes_reminders_object = UserSetting.query.filter_by(user_id=user_id,
-                                                         setting_id=10).first()
-    if notes_reminders_object:
-        notes_reminders = notes_reminders_object.value
-    else:
-        notes_reminders = None
+    settings = user.settings  # access priority, intention and notifications
```

Comment:
I realize that due to the the structure of your app and the way you have defaults setup that you may have been doing multiple queries for that. However, try to not make the same query 3 times in a row. Instead pull all of the settings and then just ask for them by name.

I didn't see anything on the models that specified anything about priority, intentions or notifications. So it seemed to me this must the name of the settings. All the more reason to make sure your code is clear about what these settings are.